tensorflow/python/eager/def_function.py

```diff
@@ -552,7 +552,10 @@ def _decorate(self, decorator):
     self._function_spec = function_lib.FunctionSpec.from_function_and_signature(
         self._python_function, self.input_signature)
 
-  def _get_tracing_count(self):
+  def experimental_get_tracing_count(self):
```

Comment:
If we're adding a new API this needs better documentation (including an example of how to use it) and a unit test.

Reply:
examples and unittests added.

Reply:
Can the example be made into a doctest as per https://www.tensorflow.org/community/contribute/docs_ref ?

Reply:
Also I don't know if users will understand what "tracing count" is. Maybe say "how many times the function has been traced"? Also mention again some conditions under which tf.function will do retraces.

Reply:
ok, will fix it shortly.

Reply:
We also need to be very clear here that people should not assume the retracing conditions are static; future versions of TF might need to retrace more or less often.